src/chart/heatmap/HeatmapView.ts

```diff
@@ -31,9 +31,10 @@ import type ContinuousModel from '../../component/visualMap/ContinuousModel';
 import { CoordinateSystem, isCoordinateSystemType } from '../../coord/CoordinateSystem';
 import { StageHandlerProgressParams, Dictionary, OptionDataValue } from '../../util/types';
 import type Cartesian2D from '../../coord/cartesian/Cartesian2D';
-import type Calendar from '../../coord/calendar/Calendar';
+import Calendar from '../../coord/calendar/Calendar';
 import { setLabelStyle, getLabelStatesModels } from '../../label/labelStyle';
 import Element from 'zrender/src/Element';
+import { RectLike, RectShape } from 'zrender';
```

Comment:
Avoid importing from the 'zrender'. It may import the whole package.

Reply:
copy that